js/screens/NotificationsScreenComponents/NotificationRow.js

```diff
@@ -231,6 +235,11 @@ class NotificationRow extends React.Component {
           </Text>
         );
         break;
+      case 800:
+        innerText = (
+          <Text style={textStyle}>{notification.data.display_username}</Text>
```

Comment:
This outputs just username (from the screenshot), should it have some text? Maybe "USERNAME is now following you".